cmd/config-downloader/downloader.go

```diff
@@ -125,26 +125,24 @@ func main() {
 	if inputConfig != "" {
 		f, err := os.Open(inputConfig)
 		if err != nil {
-			fmt.Printf("E! Failed to open Common Config: %v\n", err)
-			panic(err)
+			log.Panicf("E! Failed to open Common Config: %v", err)
```

Comment:
Did you confirm on all major OS types that these logs are visible in the agent log even though they are going to stderr instead of stdout?

Reply:
Just want to confirm this before approving